### ISSUE TITLE
Landscape mode UI improvements

### ISSUE THREAD

**Issue by sandarumk** [systers/PCSA-Android#125]
Most of the UI pages are not designed to adapt to landscape mode.

**Comment by codingblazer** [comment]
@sandarumk @rohands
I am listing all of Landscape UI related issues below ( I have tested every single screen) 

**General Issues**

1. Splash Screen - Progress bar is not visible ( which is visible in potrait)
2. Circle of Trust Screen not accommodated completely on screen as described in #102 - However I think this should not be the issue because if we accommodate circle on single screen as described in  #102, Imagebuttons will become too small for user to touch  on small devices.
3. Circle of Trust Intro slides - Partial Image of circle in 1st slide and Help Me Image overlapping with bottom navigation bar in 2nd slide as shown below -
<img src="https://cloud.githubusercontent.com/assets/13872065/20862073/de6a687c-b9c6-11e6-82fa-7da7955b6ddc.jpeg" height="" width="300">
<img src="https://cloud.githubusercontent.com/assets/13872065/20862074/df7d8cbc-b9c6-11e6-9c1a-8aa1f7273c4f.jpeg" height="" width="300">
<br>
<br>

**Alignment and Positioning Based Issues**

1. Get Help Now 
-> Contact Post Staff - Buttons not in center as described in #119
-> Contact Other Staff - Office of Victim Advocacy (same with other 2 buttons) - Heading not in center

2. Safety Tools
->Tactics of Sexual Predators - The TextView Boxes should be in center


3. Support Services
->Services after Sexual Assault -The  TextView Boxes should be in center


<img src="https://cloud.githubusercontent.com/assets/13872065/20862284/79080ef2-b9cc-11e6-99b2-e397c32de5a1.jpeg" height=""  width="300"><img src="https://cloud.githubusercontent.com/assets/13872065/20862285/7909938a-b9cc-11e6-9319-d6c92ac9148d.jpeg" height=""  width="300">

<br>

**Dialog Box Fragment Overlapping Status Bar**

1. Support Services
->Confidentiality -> Role of Staff (same with other 2 buttons) 

2. Safety Tools
->Safety Plan Basics -> Purpose of Safety Plan (same with other 2 buttons)


<img src="https://cloud.githubusercontent.com/assets/13872065/20860883/86937526-b9a8-11e6-9e0a-0c494ca4b22f.jpeg" height="" width="300" >
<img src="https://cloud.githubusercontent.com/assets/13872065/20860931/a18bfbd6-b9a9-11e6-9267-b2c308e77f80.jpeg" height="" width="300" >

<br>
<br>

**Partial UI visible and even not scrollable => Part of UI totally lost**

1. Sign Up Screen
2. Safety Tools
->SafetyPlan->Home(same with any other tab)-TextView not inside & UI not scrollable either.
->Radar ->Step 1 (same with any other step)- TextView not completey inside & UI not scrollable either.
->Safety Plan Basics - Last Button not visible.
<img src="https://cloud.githubusercontent.com/assets/13872065/20862282/69e186a6-b9cc-11e6-928b-ed08fcf09412.jpeg" height="" width="300">
<img src="https://cloud.githubusercontent.com/assets/13872065/20862281/69e1100e-b9cc-11e6-86b9-e68f65e2039a.jpeg" height="" width="300">
<br>
However, scroll of Textview works in these but that will be just workaround.

**Comment by sandarumk** [comment]
Good work. Do you want to fix these?

**Comment by codingblazer** [comment]
@sandarumk Yeah sure.

**Comment by sandarumk** [comment]
Are there any assigned issues you are currently working on? (Apart from the issues which already have PRs submitted?)

**Comment by codingblazer** [comment]
@sandarumk No, I don't have any issue assigned.

**Comment by sandarumk** [comment]
Cool. You can go ahead with this. @rohands @calistus please mention that if you have any more suggestions on the issues and improvements suggested.

**Comment by saketkumar** [comment]
@codingblazer Can i help you in this?  Can i fix the issue of Sign Up Screen? If you don't mind.

**Comment by codingblazer** [comment]
@saketkumar95 
This is a single issue because when I created #320 , I was asked to fix it under this issue only. I don't think multiple PR's for small-small parts of this issue will be good. However, if **Collaborators don't have problem with multiple PR's for this issue** , you can pick "Dialog covering status bar in Support Services"(I have already fixed the one you are asking and started working on others). But please ask @sandarumk first.

**Comment by saketkumar** [comment]
okay no problem. I just asked because there are many parts in this issue. 
@sandarumk Can i work along with @codingblazer ??

**Comment by sandarumk** [comment]
@codingblazer if you are willing to work together, there is no problem from our side. You can converse here and distribute the work load.

**Comment by codingblazer** [comment]
@sandarumk Cool. @saketkumar95 Do you still want to work ?

**Comment by saketkumar** [comment]
@codingblazer  Yeah. Tell me what else are remaining in this issue.

**Comment by codingblazer** [comment]
@saketkumar95 Great !! You can start with "Dialog Box Covering Status Bar in Support Services' (for all the three buttons). Ask me if you need any help.

**Comment by saketkumar** [comment]
Yeah. Sure. Thanks @codingblazer :)

**Comment by sandarumk** [comment]
Any update on this?

**Comment by codingblazer** [comment]
@sandarumk I am really sorry for delay. I got tied up in exams. I am done now with 80% of the work. Will send PR by tomorrow for sure.

**Comment by codingblazer** [comment]
@saketkumar95 Are you done with them.

**Comment by saketkumar** [comment]
Yes done. Will be sending the PR.

**Comment by codingblazer** [comment]
@saketkumar95 have you fixed it for support services or for safety tools too.

**Comment by BaaniLeen** [comment]
@codingblazer @saketkumar95 I would like to contribute in making the landscape version for Support Services or Safety Tools if they are still left.

**Comment by codingblazer** [comment]
Hey @BaaniLeen, it will be great if we ask @saketkumar95 first as he was working on this.
But since its been long since he replied , you can ask @sandarumk .

**Comment by BaaniLeen** [comment]
@sandarumk Shall we wait or shall I start working on the landscape version for Support Services or Safety Tools since @saketkumar95 has not replied for long?

**Comment by BaaniLeen** [comment]
@sandarumk Shall we wait or shall I start working on the landscape version for Support Services or Safety Tools since @saketkumar95 has not replied for long?

**Comment by saketkumar** [comment]
@BaaniLeen I'm done with this but some errors are there. For reference you can check on my repo and you can go ahead to work on this. I'm currently busy with my university exams. :)

**Comment by sandarumk** [comment]
@BaaniLeen I think two people are enough for this issue. Let @saketkumar95 finish what he is doing.

**Comment by BaaniLeen** [comment]
Sure..! No problem :)

**Comment by saketkumar** [comment]
@sandarumk Sure. I will send the PR soon after my university exam ends. Apologize for the delay. :)

**Comment by HimanshuS01** [comment]
Hey @saketkumar95 Are you still working on this ? As it's been a month this issue is still pending.
@sandarumk @codingblazer I would like to work on **Dialog Box Covering Status Bar in Support Services**. Can I proceed If you allow me to do so ?  Thanks :)

**Comment by codingblazer** [comment]
@HimanshuS01 Yes, it's been too long since this is pending and issue also need a simple fix. You may proceed if @sandarumk approve.

**Comment by HimanshuS01** [comment]
@sandarumk I would like to work on this.
can I send the PR?
_Thank you_

**Comment by sandarumk** [comment]
@saketkumar95 Any updates? Are you still on this?

**Comment by HimanshuS01** [comment]
@sandarumk , as there are no updates from the @saketkumar95 's side and one part of this issue is still pending. 
@sandarumk can I proceed with this issue now and send the PR?
thanks :)

**Comment by HimanshuS01** [comment]
@rohands  I have no assigned issue as of now. Can I work on this ? 
Thank You
